### PR TITLE
shell.info: respect shell verbosity

### LINKED ISSUE #138: Minor (cosmetic) - could `Loading save analysis` messages be suppressed?
Those are definitely useful as verbose/debug info but most of the time, why would the user care? Could they be hidden behind a verbose flag or even easier, and env var? (e.g. `CARGO_UDEPS_VERBOSE`)

Asking because the typical use case is...

```
$ cargo +nightly udeps

<2 pages of 'info: Loading save analysis from ...'>
All deps seem to have been used.
```

(apologies in advance if I'm missing something obvious here)

### PR DESCRIPTION
**Pull request by utkarshgupta137**
Fixes #138

### CODE REVIEW

**Comment by qryxip** [comment]
I think `Shell` [already contains the `--quiet` flag](https://docs.rs/cargo/0.67.1/cargo/core/shell/struct.Shell.html#method.verbosity).

Seeing [`Shell::warn` implementation](https://docs.rs/cargo/0.67.1/src/cargo/core/shell.rs.html#260-265), It seems I wrote `<Shell as crate::ShellExt>::info` wrongly.

**Comment by utkarshgupta137** [comment]
> I think `Shell` [already contains the `--quiet` flag](https://docs.rs/cargo/0.67.1/cargo/core/shell/struct.Shell.html#method.verbosity).
> 
> Seeing [`Shell::warn` implementation](https://docs.rs/cargo/0.67.1/src/cargo/core/shell.rs.html#260-265), It seems I wrote `<Shell as crate::ShellExt>::info` wrongly.

I somehow missed the verbosity method. That simplifies it a lot!

File: src/lib.rs [hunk 1]
```diff
@@ -1163,17 +1163,20 @@ trait ShellExt {
...
-			format!(
-				"{} {}\n",
-				if self.err_supports_color() {
-					Colour::Cyan.bold().paint("info:").to_string()
-				} else {
-					"info:".to_owned()
-				},
-				message,
+		match self.verbosity() {
+			Verbosity::Quiet => Ok(()),
```
Comment (est31) [inline src/lib.rs:1167]: Could this be an `if let` with a `return` inside?

**Review by est31 — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (est31) [reply src/lib.rs:1167]: Nevermind it's not that important. Merging.

**Comment by utkarshgupta137** [comment]
@est31 When can we expect a release?

**Comment by est31** [comment]
@utkarshgupta137 see #151

**Pull request merged**